src/oc-client.js

```diff
@@ -520,7 +520,11 @@ var oc = oc || {};
 
         oc.renderByHref($component.attr('href'), function (err, data) {
           if (err || !data) {
-            $component.html('');
+            $component
+              .attr('data-rendering', 'false')
+              .attr('data-rendered', 'false')
+              .html('');
+            oc.events.fire('oc:failed', { originalError: err, data: data });
```

Comment:
should we add a link to component in payload as well?

Reply:
added